doc/user/how-to/metadata.md

```diff
@@ -38,3 +41,8 @@ Open the **provenance.txt** file to see all of the processing steps applied to
 the data for QGreenland.
 
 ![layer_dir.png](/_images/layer_dir.png)
+
+```{note}
+To see the layer's geospatial metadata (e.g., the coordinate reference system
+and spatial extent), we recommend [using QGIS](#via-qgis-layer-properties).
```

Comment:
These anchor links can be broken easily if someone changes the title of the section. If we use an explicit cross-reference, these links are more durable and can throw errors at build time if we break either end of the crossref!

https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#explicit-targets

Reply:
Nice, I'll take a look at this! Note that our build process already produces an error if one of these links is broken (at least within single pages)

Reply:
Oh cool, I thought it just skipped checks for this style of link.

Reply:
I think it's just http links that get skipped. I replaced a bunch with references `#like-this` in other parts of this PR. In a couple of cases I changed a title without the `#like-this` link, and our build process thew an error indicating where those broken links were.

Reply:
Issue to address this more holistically: https://github.com/nsidc/qgreenland/issues/723

Reply:
Nice!